examples/c++/tmp006.cxx

```diff
@@ -58,8 +58,8 @@ int main(int argc, char **argv)
   // update and print available values every second
   while (run)
     {
-      // Print out temperature value in °C
-      std::cout << "Temperature: " << mySensor->getTemperature(true) << " °C"
+      // Print out temperature value in *C
+      std::cout << "Temperature: " << mySensor->getTemperature(true) << " *C"
```

Comment:
Hmm, this "_Major_" logging rule should probably be disabled :)  Even the [justifications](https://sonarcloud.io/organizations/default/rules#rule_key=cpp%3AS106) for the rule are generally bogus, IMO.